lib/SyTest/Federation/Client.pm

```diff
@@ -240,6 +240,8 @@ sub join_room
             auth_events content depth prev_events room_id sender
             state_key type ) ),
 
+         # The spec does not require the origin field, but Dendrite does. This can be removed
```

Comment:
@richvdh Is this similar to what you were expecting? Let me know if you want me to change the message. Thanks!